Free-Code-Camp-Full-Stack-Development-Certificate.md

```diff
@@ -4,8 +4,11 @@ In order to participate in our nonprofit projects, you first need to earn our Fu
 
 ### Here's how to earn our verified Full Stack Development Certificate:
 
-1. Complete all our current Algorithm Challenges
-1. Complete all our current Front End Project Challenges
-1. Complete all our current Back End Project Challenges
+1. Earn our current Front End Development Cerificate
+2. Earn our current Data Visualization Cerificate
+3. Earn our current Back End Development Cerificate
+4. Complete all our current Greenfield Nonprofit Projects
+5. Complete all our current Legacy Code Nonprofit Projects
 
-Once you've completed all of the above, you can complete our "Claim your Full Stack Development Certificate" challenge, which will require you to sign our Academic Honesty Pledge. Then we will instantly issue your verified certificate, which will become accessible from your portfolio page.
\ No newline at end of file
+
+Once you've completed all of the above, you can complete our "Claim your Full Stack Development Certificate" challenge, which will require you to sign our Academic Honesty Pledge. Then we will instantly issue your verified certificate, which will become accessible from your portfolio page.

```

Comment:
Can you please provide the link for Full Stack Development Certification. Mine is showing locked